src/styles/globals.css

```
@@ -1155,3 +1155,16 @@
+
+/* Pulsing dot used by active-timer indicators (sidebar widget, action surfaces,
+   calendar TimeEntry blocks). Module-scoped @keyframes elsewhere are not reachable
+   from these surfaces, so the canonical definition lives here. */
+@keyframes pulse {
+  0%,
+  100% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.4;
+  }
```

Stylelint (17.11.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)